### PR TITLE
[MW] Improve landing screen

### PR DESCRIPTION
**Pull request by mvaivre**
Hopefully faster and more beautiful landing screen.

### CODE REVIEW

**Comment by mvaivre** [comment]
TODO: fix bottom padding on Android
image

**Review by nop33 — COMMENTED** [review]
Code looks good, and it works well on the simulators! Just one comment about the image :)

**Review by nop33 — COMMENTED** [review]
👏 Looks amazing

As mentioned on Signal, Android landing screen is missing the bg colors in the simulator...

**Review by nop33 — APPROVED** [review]

File: apps/mobile-wallet/src/navigation/RootStackNavigation.tsx [hunk 1]
```diff
@@ -247,5 +246,5 @@ const AppUnlockHandler = ({ children }: { children: ReactNode }) => {
     walletMnemonic
   ])
 
-  return children
+  return null
```
[the remark this answers is not in the document]
Comment (nop33) [reply apps/mobile-wallet/src/navigation/RootStackNavigation.tsx:249]: @mvaivre Could you add a TODO here or create an issue? Thanks!

**Comment by nop33** [comment]
Changed base branch to `next`

**Pull request merged**